wormhole-connect/src/views/Bridge/TransferLimitedWarning.tsx

```diff
@@ -1,25 +1,228 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { ChainName } from '@wormhole-foundation/wormhole-connect-sdk';
 import useIsTransferLimited from 'hooks/useIsTransferLimited';
-import AlertBanner from 'components/AlertBanner';
 import { wh } from 'utils/sdk';
+import { Dialog, ScopedCssBaseline } from '@mui/material';
```

Comment:
to allow three seaking would be good to import using:

import Dialog 
```suggestion
import Dialog from '@mui/material/Dialog';
import ScopedCssBaseline from '@mui/material/ScopedCssBaseline';

```

Reply:
Done!